template/package.json

```diff
@@ -141,7 +141,7 @@
     "bundler": "rollup -c",
     "bundler:test": "rollup -c -w",
     "scssLint": "stylelint \"./src/**/*.scss\"",
-    "dev": "concurrently --kill-others 'npm run build:watch' 'npm run serve'",
+    "dev": "concurrently --kill-others \"npm run build:watch\" \"npm run serve\"",
```

Comment:
Why was this change necessary? The single quotes worked fine - why escape the double quote?

Reply:
See the linked issue 

- https://github.com/AlaskaAirlines/WC-Generator/issues/274